### PR TITLE
Make prometheusRule.interval default to 1m

### PR DESCRIPTION
**Pull request by danksim**
#### Which component this PR applies to?

helm-charts

#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
What: Set default `prometheusRule.interval` to `1m` instead of `global. evaluation_interval` (which has a default value of `1m`)
Why: Helm release was failing since `prometheusRule.interval` was defaulting to `nil` when not explicitly set

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Please close if necessary

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by gjtempleton** [comment]
Hey, thanks for the PR, we use helm-docs to generate part of the `README.md` from the values file, you'll need to update the default value and comment there. [Pre-commit](https://github.com/kubernetes/autoscaler/tree/master/charts#pre-commit-hooks) should then take care of running the helm-docs command to update the README.

**Comment by gjtempleton** [comment]
Thanks for that, one last request, can you squash your commits down to one please and then we'll be good to merge.

**Comment by danksim** [comment]
Just squashed and pushed. Thank you! @gjtempleton

File: charts/cluster-autoscaler/values.yaml [hunk 1]
```diff
@@ -358,8 +358,8 @@ prometheusRule:
   additionalLabels: {}
   # prometheusRule.namespace -- Namespace which Prometheus is running in.
   namespace: monitoring
-  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to `global.evaluation_interval` if not set).
-  interval: null
+  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the default value of `global.evaluation_interval` (which is 1m) if not set).
```
Comment (gjtempleton) [inline charts/cluster-autoscaler/values.yaml:361]: Related to above comment on emitting the interval if not set.
```suggestion
  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the Prometheus level value of `global.evaluation_interval` if not set).
```

File: charts/cluster-autoscaler/values.yaml [hunk 2]
```diff
@@ -358,8 +358,8 @@ prometheusRule:
   additionalLabels: {}
   # prometheusRule.namespace -- Namespace which Prometheus is running in.
   namespace: monitoring
-  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to `global.evaluation_interval` if not set).
-  interval: null
+  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the Prometheus level value of `global.evaluation_interval` if not set).
+  interval: nil
```
Comment (gjtempleton) [inline charts/cluster-autoscaler/values.yaml:362]: I don't think this change is necessary, templating with this as the default value:
```yaml
spec:
  groups:
    - name: release-name-aws-cluster-autoscaler
      interval: nil
      rules:
        []
```

Templating with the new logic, but the old `interval: null`:
```yaml
spec:
  groups:
    - name: release-name-aws-cluster-autoscaler
      rules:
        []
```

**Comment by gjtempleton** [comment]
Closing for now, please re-open if you're able to address the feedback and conflicts.
/close

**Pull request closed without merging**